marketplacetests/marketplace/regions/search_results.py

```diff
@@ -25,6 +23,12 @@ def search_results(self):
         search_results = self.marionette.find_elements(*self._search_result_locator)
         return [Result(self.marionette, result) for result in search_results]
 
+    def install_searched_app(self, search_term):
```

Comment:
I agree with that, but I also think it is better for us to have coverage with both installing methods.
Users will search for the apps and install them directly from the search page.
Do you agree with having both?